Extension/src/LanguageServer/configurations.ts

```diff
@@ -736,34 +737,57 @@ export class CppProperties {
     }
 
     private resolveAndSplit(paths: string[] | undefined, defaultValue: string[] | undefined, env: Environment, glob = false): string[] {
```

Comment:
Existing code, but could you add the type to this arg?  I'm not sure why lint isn't reporting an issue that it's missing a type.

Reply:
I think Garrett's change may have disabled the typing requirement, but I think that's was desired, i.e. like auto in C++.